### PR TITLE
updated data caching

### PR DESCRIPTION
**Pull request by firmanJS**
## OVERVIEW

cache setup log and tracing

### CODE REVIEW

File: helpers/email/index.js [hunk 1]
```diff
@@ -39,28 +39,22 @@ const optionsWithAttachment = (subject, attachments, email, jobName) => {
...
-    job_status: null, job_progress: 100,
-    type: 'email', message: null
-  }
+const condition = async (err, path, jobId, res) => {
   if(err) {
     console.info(`sending email error : ${err}`)
-    param.job_status = 'Error'
-    param.message = err.toString()
-    await updateLogJob(jobId, param)
+    const set = { 'message.email':err.toString(), 'job_status': 'Error', 'job_progress': 50 }
```
Comment (sandisunandar99) [inline helpers/email/index.js:45]: why status error show when job progress 50%? why not to back 0% if got to error?

**Review by sandisunandar99 — COMMENTED** [review]
@firmanJS check comment

File: helpers/email/index.js (hunk 1 above)
Comment (firmanJS) [reply helpers/email/index.js:45]: I think set 50% i user failed this is stuck progress, but nice feedback dude i will set to 0%

**Comment by firmanJS** [comment]
up guys @khihadysucahyo @sandisunandar99

File: helpers/email/index.js (hunk 1 above)
Comment (sandisunandar99) [reply helpers/email/index.js]: thnks

**Pull request merged**